### DISCUSSION TITLE
How to handle excel generation with more than 1,048,576 rows

### DISCUSSION THREAD

**Discussion by ankurkumarHL** [jxlsteam/jxls#190] [Q&A] [answered]
Hi, We have cases where we need to generate excel with more than 1,048,576 rows. We have tested is and jxls just ignored all the rows more than 1,048,576. Is there any way to create new sheet where a sheet is overflow. How to handle this?

**Comment by SoltauFintel** [comment, marked as answer]
1M rows is the limit of Microsoft Excel.

**Reply by ankurkumarHL** [reply]
@SoltauFintel is there any support in jxls in which if records overflow a sheet then it will create a new sheet in the same excel and populate rest of the records in new sheet and so on?

**Reply by SoltauFintel** [reply]
Please have a look at http://jxls.sourceforge.net/reference/multi_sheets.html and/or clone our repo and look for examples.

You must limit the lists. Jxls does it not for you.
